sos/report/plugins/logs.py

```diff
@@ -105,4 +105,18 @@ def postproc(self):
             r"pwd=[******]"
         )
 
+
+class CosLogs(Logs, CosPlugin):
+    option_list = [
+        PluginOpt("log_days", "the number of days logs to collect", "", 3)
+    ]
```

Comment:
Same as above, the parameters for `PluginOpt` do not match the previous tuple format.

Reply:
makes sense.